bots/zephyr_mirror_backend.py

```diff
@@ -105,7 +105,7 @@ def unwrap_lines(body):
         elif (line == "" or
             previous_line == "" or
             re.match(r'^\W', line, flags=re.UNICODE) or
-            different_paragraph(previous_line, line)):
+                different_paragraph(previous_line, line)):
```

Comment:
I think actually the previous several lines should be indented like this to make the code as clear as possible:

```
         elif (line == "" or
  ￼             previous_line == "" or
￼               re.match(r'^\W', line, flags=re.UNICODE) or
               different_paragraph(previous_line, line)):
```

(with the statements nicely aligned)

There's a few other instances with the same problem.